packages/react-core/src/components/Pagination/examples/Pagination.md

```diff
@@ -252,3 +259,87 @@ class PaginationTop extends React.Component {
   }
 }
 ```
+
+### Sticky (top)
```

Comment:
Sorry to change my mind, but I'm going to recommend similar to #5000 that this be made a fullscreen example with a toggle for top/bottom with a `<div style={{ height: '100vh' }} />` item so that it demos well!

Reply:
Updated the example for fullscreen with a toggle, but for some reason the pagination isn't moving to the bottom despite the div styling (but it's css looks correct). Did I set up the example incorrectly?

Reply:
Sticky doesn't behave like absolute positioning - it depends on where the element is on the page in relation to other things. Absolute positioning would just move it wherever you tell it to go. I think you'll need top and bottom examples, unless your toggle also moves the pagination element in the DOM. I think it would help to have a lot of text on the example page, too, to see how the sticky position behaves when the user scrolls.

Reply:
Oh I see, thanks @mcoker! I've update the example so it toggles properly, showing a card gallery to illustrate the sticky aspect.